remotehttp.go

```diff
@@ -119,6 +120,14 @@ func (r *RemoteHTTPBase) IssueHttpRequest(method string, u *url.URL, getReader G
 	if r.opt.HTTPAuth != "" {
 		req.Header.Set("Authorization", r.opt.HTTPAuth)
 	}
+	if r.opt.HTTPCookie != "" {
+		req.Header.Set("Cookie", r.opt.HTTPCookie)
+	}
+	// Set from env var if appropriate
+	httpCookie := os.Getenv("DESYNC_HTTP_COOKIE")
```

Comment:
This basically means the env var takes precedence over the config file. It might be good to swap the logic around.

Reply:
Good shout, will change that.

Reply:
In fact, I'll remove it. The previous `DESYNC_HTTP_AUTH` header is only used for the server. I could add it to the server, but not sure how much sense that makes!